social-network/src/YourArticles.js

```diff
@@ -0,0 +1,48 @@
+// yourArticles.js
+import React from 'react';
+import yourArticlesJson from './your-articles.json';
+
+class yourArticlesComp extends React.Component {
+  
+    render() {
+        const articles = [];
+        for (var i=0; i < yourArticlesJson.length; i++) {
+          articles.push(yourArticlesJson[i]);
+        }
+        const articlesMod = articles.map(array => { 
+          array = { ...array, postedDate: array.postedDate.substring(0, 10) };
+          return array;
+        });
+
+        function Articles(props) {  
```

Comment:
Since props are being used in this component and the component above, `propTypes` are needed.